test/e2e-tests.sh

```diff
@@ -14,15 +14,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export GO111MODULE=on
 source $(dirname $0)/../vendor/github.com/tektoncd/plumbing/scripts/e2e-tests.sh
 
+# Namespace used for tests
+readonly TEST_NAMESPACE="operator-tests"
+
+function tekton_setup() {
+  header "Installing Tekton Operator"
+  kubectl create namespace $TEST_NAMESPACE
+  kubectl apply -n $TEST_NAMESPACE -f deploy/crds/
```

Comment:
why do we need `-n $TEST_NAMESPACE` while applying CRDs as CRDs are clusterwide

Reply:
or should it be ` kubectl apply -n $TEST_NAMESPACE -f deploy/crs/`

Reply:
Since I create the CRDs available under config/, I can use kubectl apply -n $TEST_NAMESPACE -f config/ to install everything, including the crds and the deployment. So I can drop this line.